miso-web/src/main/webapp/scripts/list_storage_location.js

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright (c) 2012. The Genome Analysis Centre, Norwich, UK
+ * MISO project contacts: Robert Davey @ TGAC
+ * *********************************************************************
+ *
+ * This file is part of MISO.
+ *
+ * MISO is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * MISO is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with MISO.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * *********************************************************************
+ */
+
+ListTarget.storage_location = {
+  name: "Storage Locations",
+  createUrl: function(config, projectId) {
+    throw "Storage locations must be specified statically.";
+  },
+  queryUrl: null,
+  createBulkActions: function(config, projectId) {
+    return [];
+  },
+  createStaticActions: function(config, projectId) {
+    switch (config.slug) {
+    case 'rooms':
+      return [{
+        "name": "Add",
+        "handler": Freezer.addRoom
```

Comment:
After adding a room, the "Room added" message pops up but the table is still empty. How about force page refresh after calling `addRoom`?

Reply:
We don't have a unique constraint on room names? :scream: